cli/src/lib/block-explorer-client.ts

```diff
@@ -148,6 +153,48 @@ export class BlockExplorerClient implements BlockExplorer {
     }
   }
 
+  async getLogs(
```

Comment:
I ended up not using this for now, but I'm sure that it's going to be needed for real production.